src/storage/slotted_page.c

```diff
@@ -4011,6 +4011,27 @@ spage_get_record_type (PAGE_PTR page_p, PGSLOTID slot_id)
   return slot_p->record_type;
 }
 
+bool
+spage_is_record_deleted (PAGE_PTR page_p, PGSLOTID slot_id)
```

Comment:
Nit:

Too general function name. However, it cannot use in the reuse-oid table.

Reply:
this function is removed